Formula/p/patch-package.rb

```diff
@@ -0,0 +1,35 @@
+require "language/node"
+
+class PatchPackage < Formula
+  desc "Writes your git commit messages for you with AI"
+  homepage "https://github.com/Nutlope/patch-package"
```

Comment:
These don't seem right. The homepage 404's and the npm page states this "lets app authors instantly make and keep fixes to npm dependencies". Should probably be https://github.com/ds300/patch-package

Reply:
yeah, that is right, fixed it now.